src/app/games-page/games-page.component.html

```diff
@@ -1 +1,43 @@
-<p>games-page works!</p>
+<div class="cards-container"
+infiniteScroll
+[infiniteScrollDistance]=exploreScrollParams.infiniteScrollDistance
+[infiniteScrollThrottle]=exploreScrollParams.infiniteScrollThrottle
+(scrolled)="onScroll()"
+>
+    <div *ngFor="let game of gamesList">
+        <div class="game-card">
```

Comment:
Duplication of game card, extract this to a separate component and use in"Explore" and "Favorites" pages